checks/code_review_test.go

```diff
@@ -0,0 +1,247 @@
+				Score: -1,
+			},
+		},
+		{
+			name: "no PR's with error",
+			err:  errors.New("error"),
+			expected: checker.CheckResult{
+				Score: -1,
+			},
+		},
+		{
+			name:      "no PR's with error as well as commits",
+			err:       errors.New("error"),
+			commiterr: errors.New("error"),
+			expected: checker.CheckResult{
+				Score: -1,
+			},
+		},
+		{
+			name: "Valid PR's and commits as not a bot",
+			prs: []clients.PullRequest{
+				{
+					Number:   1,
+					MergedAt: time.Now(),
+					Reviews: []clients.Review{
+						{
+							State: "APPROVED",
+						},
+					},
+					Labels: []clients.Label{
+						{
+							Name: "lgtm",
+						},
+					},
+				},
+			},
+			commits: []clients.Commit{
+				{
+					SHA: "sha",
+					Committer: clients.User{
+						Login: "user",
+					},
+				},
+			},
+
+			expected: checker.CheckResult{
+				Score: 10,
+				Pass:  true,
+			},
+		},
+
+		{
+			name: "Valid PR's and commits as bot",
+			prs: []clients.PullRequest{
+				{
+					Number:   1,
+					MergedAt: time.Now(),
+					Reviews: []clients.Review{
+						{
+							State: "APPROVED",
+						},
+					},
+					Labels: []clients.Label{
+						{
+							Name: "lgtm",
+						},
+					},
+				},
+			},
+			commits: []clients.Commit{
+				{
+					SHA: "sha",
+					Committer: clients.User{
+						Login: "bot",
+					},
+				},
+			},
+
+			expected: checker.CheckResult{
+				Score: 10,
+				Pass:  true,
+			},
+		},
+
+		{
+			name: "Valid PR's and commits not yet merged",
+			prs: []clients.PullRequest{
+				{
+					Number: 1,
+					Reviews: []clients.Review{
+						{
+							State: "APPROVED",
+						},
+					},
+					Labels: []clients.Label{
+						{
+							Name: "lgtm",
+						},
+					},
+				},
+			},
+			commits: []clients.Commit{
+				{
+					SHA: "sha",
+					Committer: clients.User{
+						Login: "bot",
+					},
+				},
+			},
+
+			expected: checker.CheckResult{
+				Score: -1,
+			},
+		},
+		{
+			name: "Valid PR's and commits with merged by someone else",
+			prs: []clients.PullRequest{
+				{
+					Number: 1,
+					Reviews: []clients.Review{
+						{
+							State: "APPROVED",
+						},
+					},
+					Labels: []clients.Label{
+						{
+							Name: "lgtm",
+						},
+					},
+					MergeCommit: clients.Commit{
+						SHA: "sha",
+						Committer: clients.User{
+							Login: "bob",
+						},
+					},
+				},
+			},
+			commits: []clients.Commit{
+				{
+					SHA: "sha",
+					Committer: clients.User{
+						Login: "bot",
+					},
+				},
+			},
+
+			expected: checker.CheckResult{
+				Score: -1,
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		tt := tt // Re-initializing variable so it is not changed while executing the closure below
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			ctrl := gomock.NewController(t)
+			mockRepo := mockrepo.NewMockRepoClient(ctrl)
+			mockRepo.EXPECT().ListMergedPRs().DoAndReturn(func() ([]clients.PullRequest, error) {
+				if tt.err != nil {
+					return nil, tt.err
+				}
```

Comment:
Can simply `return tt.prs, tt.err` here? `if` is not needed AFAICT.

Reply:
Thanks!